.travis.yml

```diff
@@ -3,7 +3,7 @@ lein: 2.8.1
 dist: trusty
 jdk:
   - oraclejdk8
-  - oraclejdk10
+  - openjdk10
```

Comment:
FYI @justinstoller @Magisus  oraclejdk10 is no more, it eol'd in september and its downloads are no longer available.